miso-web/src/main/webapp/scripts/tile.js

```diff
@@ -0,0 +1,140 @@
+    div.appendChild(title);
+
+    var content = document.createElement("DIV");
+    content.setAttribute("class", "widget ui-corner-bottom");
+    content.appendChild(inner);
+    div.appendChild(content);
+  };
+
+  return {
+    title: function(label, status) {
+      var title = document.createElement("DIV");
+      title.setAttribute("class", "name");
+      title.setAttribute("style", "font-weight:bold");
+      title.innerText = label;
+      title.appendChild(status);
+      return title;
+    },
+    error: function(message) {
+      var errorP = document.createElement("P");
+      errorP.setAttribute("class", "parsley-error");
+      errorP.innerText = "⚠ " + message;
+      return errorP;
+    },
+    lines: function(lines, special) {
+      var p = document.createElement("P");
+      if (special) {
+        p.setAttribute("style", "font-style:italic");
+      }
+
+      lines.filter(function(line) {
+        return !!line;
+      }).forEach(function(line, index, arr) {
+        p.appendChild(document.createTextNode(line));
+        if (index < arr.length - 1) {
+          p.appendChild(document.createElement("BR"));
+        }
+      });
+      return p;
+    },
+    statusOk: function() {
+      var status = document.createElement("SPAN");
+      status.setAttribute("style", "color:#66ce16");
+      status.innerText = "⬤";
+      return status;
+    },
+    statusBad: function() {
+      var status = document.createElement("SPAN");
+      status.setAttribute("style", "color:#db0f34");
+      status.innerText = "⬛";
+      return status;
+    },
+    statusBusy: function() {
+      var status = document.createElement("SPAN");
+      status.setAttribute("style", "color:#4286f4");
+      status.innerText = "🞜";
+      return status;
+    },
+    make: function(tileparts, clickHandler) {
+      var div = document.createElement("DIV");
+      div.setAttribute("class", "tile");
+      tileparts.forEach(function(part) {
+        div.appendChild(part);
+      })
+      div.onclick = clickHandler;
+      return div;
+    },
+    createPane: function(elementId, target, items) {
+      var innerContent = document.createElement("DIV");
+      items.map(target.transform).forEach(function(item) {
+        if (item) {
+          innerContent.appendChild(item);
+        }
+      });
+      showPane(elementId, target, innerContent);
+    },
+    createPaneAjax: function(elementId, target, url) {
+      var loader = document.createElement("IMG");
+      loader.src = "/styles/images/ajax-loader.gif";
+      showPane(elementId, target, loader);
+
+      jQuery.ajax({
+        'dataType': 'json',
+        'type': 'GET',
+        'url': url,
+        'contentType': 'application/json; charset=utf8',
+        'success': function(data, textStatus, xhr) {
+          Tile.createPane(elementId, target, data);
+        },
+        'error': function(xhr, textStatus, errorThrown) {
+          var lines = ['Error: ' + errorThrown];
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'lines' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AUnusedVariable)